configs/default/AFNG-ALIENFLIGHTF4.config

```diff
@@ -84,71 +79,69 @@ timer C08 AF3
-resource MOTOR 8 C09
-resource PWM 1 A08
-resource PWM 2 B00
-resource PWM 3 B01
-resource PWM 4 B14
-resource PWM 5 B15
-resource LED_STRIP 1 B15
-resource PPM 1 A08
-
-# DMA
-dma SPI_TX 2 0   # SPI_TX 2: DMA1 Stream 4 Channel 0
-dma ADC 1 0      # ADC 1:    DMA2 Stream 0 Channel 0
-dma pin A08 1    # pin A08:  DMA2 Stream 1 Channel 6
-dma pin B00 0    # pin B00:  DMA1 Stream 7 Channel 5
-dma pin B01 0    # pin B01:  DMA1 Stream 2 Channel 5
-dma pin B14 1    # pin B14:  DMA2 Stream 2 Channel 6
-dma pin B15 0    # pin B15:  DMA2 Stream 6 Channel 0
-dma pin B08 0    # pin B08:  DMA1 Stream 7 Channel 2
-dma pin A00 0    # pin A00:  DMA1 Stream 2 Channel 6
-dma pin A01 0    # pin A01:  DMA1 Stream 4 Channel 6
-dma pin C06 0    # pin C06:  DMA2 Stream 2 Channel 0
-dma pin C07 0    # pin C07:  DMA2 Stream 2 Channel 0
-dma pin C08 1    # pin C08:  DMA2 Stream 4 Channel 7
-dma pin C09 0    # pin C09:  DMA2 Stream 7 Channel 7
-
-# DMA stream conflict if burst mode is not used
-set dshot_burst = ON
-
-# Serial ports
-resource SERIAL_TX 1 A09
-resource SERIAL_RX 1 A10
-
-resource SERIAL_TX 2 A02
-resource SERIAL_RX 2 A03
-resource INVERTER 2 C15
-
-resource SERIAL_TX 4 C10
-resource SERIAL_RX 4 C11
-
-# ADC
-resource ADC_BATT 1 C00
-resource ADC_RSSI 1 C04
-resource ADC_CURR 1 C01
-resource ADC_EXT 1 C05
-
-# Remaining
-resource RX_BIND_PLUG 1 B02
-resource ESCSERIAL 1 A08
 
-# Some configs
+# dma
+dma SPI_TX 2 0
+# SPI_TX 2: DMA1 Stream 4 Channel 0
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A08 1
+# pin A08: DMA2 Stream 1 Channel 6
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B14 1
+# pin B14: DMA2 Stream 2 Channel 6
+dma pin B15 0
+# pin B15: DMA2 Stream 6 Channel 0
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A01 0
+# pin A01: DMA1 Stream 4 Channel 6
+dma pin C06 0
+# pin C06: DMA2 Stream 2 Channel 0
+dma pin C07 0
+# pin C07: DMA2 Stream 2 Channel 0
+dma pin C08 1
+# pin C08: DMA2 Stream 4 Channel 7
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+# feature
+feature -RX_PARALLEL_PWM
 feature RX_SERIAL
 feature MOTOR_STOP
+
+# map
 map TAER1234
-serial 0 0 115200 57600 0 115200
+
+# serial
 serial 1 64 115200 57600 0 115200
-serial 3 0 115200 57600 0 115200
+
+# master
+set align_mag = CW180FLIP
+set mag_align_pitch = 1800
+set mag_align_yaw = 1800
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
 set serialrx_provider = SPEK2048
-set spektrum_sat_bind = 5
+set spektrum_sat_bind = 9
 set blackbox_device = SDCARD
-set adc_device = 1
+set dshot_burst = ON
 set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set sdcard_detect_inverted = ON
+set sdcard_mode = SPI
+set sdcard_spi_bus = 2
+set flash_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
+set gyro_1_align_yaw = 2700
+set i2c1_pullup = ON
```

Comment:
Looks like the defaults for the current scale/offset defined in the target are not included.
```
#define CURRENT_METER_OFFSET_DEFAULT 2500                      // ACS712/714-30A - 0A = 2.5V
#define CURRENT_METER_SCALE_DEFAULT -667                       // ACS712/714-30A - 66.666 mV/A inverted mode
```

Reply:
We never finished development of Version 2 variants with the current sensor as we had planed and prototyped for a while. We progressed to the STM32F7. So this settings are obsolete.